darts/backtesting/backtesting.py

```diff
@@ -365,15 +365,16 @@ def backtest_gridsearch(model_class: type,
 
     Parameters
     ----------
-    model
+    model_class
         The ForecastingModel subclass to be tuned for 'series'.
     parameters
         A dictionary containing as keys hyperparameter names, and as values lists of values for the
         respective hyperparameter.
     train_series
         The univariate TimeSeries instance used for training (and also validation in split mode).
-    test_series
+    val_series
         The univariate TimeSeries instance used for validation in split mode.
+        Use `train` to compare with model.fitted_values
```

Comment:
`model.fitted_values` are the values obtain when fitting a model (ExponentialSmoothing in our case) on a time series, the result of the fit (a TimeSeries with same index as the train time series). It is a quick way to assess the model (we don't have to compute the predictions each time), but I understand that we cannot see if the fitted model underfit or overfit and will give bad predictions.
It is a trade off between computation time and efficiency. Thus, it can be dropped if you think it is not a good thing to have.
I mainly used this version for the M4 benchmark because the original script used this method to select the best model.